### PR TITLE
[pt] Temporarily patch page to avoid link-check failure

### PR DESCRIPTION
**Pull request by emdneto**
Address #5423

### CODE REVIEW

**Review by edsoncelio — APPROVED** [review]

**Review by EzzioMoreira — APPROVED** [review]

File: content/pt/docs/concepts/instrumentation/libraries.md
Comment (svrnm) [inline content/pt/docs/concepts/instrumentation/libraries.md:7]: I think you can drop this attribute if it is empty, cc @chalin

Comment (chalin) [reply content/pt/docs/concepts/instrumentation/libraries.md:7]: Yes. Drop the `htmltest` entry entirely.

Comment (chalin) [reply content/pt/docs/concepts/instrumentation/libraries.md:7]: There's a broader issue to be addressed though. I'll comment separately from this thread.

**Review by chalin — CHANGES_REQUESTED** [review]
The intent of #5423 is remind the two named locales to catch up with the updates made to the Java pages, not by eliminating the link, but by catching up with the `en` versions of the pages.

By making the change to the page without updating the `a570a00c7a238ffe26528d7bfb20efdbaf939c39`, you are breaking the correspondence.

**Comment by chalin** [comment]
> The intent of #5423 is remind the two named locales to catch up with the updates made to the Java pages, not by eliminating the link, but by catching up with the `en` versions of the pages.
> 
> By making the change to the page without updating the `a570a00c7a238ffe26528d7bfb20efdbaf939c39`, you are breaking the correspondence.

@open-telemetry/docs-maintainers - we should decide if the "quick fix" in this PR, which introduces a drift/deviation from `a570a00c7a238ffe26528d7bfb20efdbaf939c39` is an acceptable deviation. I would prefer not, but given that there are more significant changes in the page, it might be an acceptable "temporary" patch so that we get restore full link checking. Thoughts on setting this precedent?

**Review by chalin — APPROVED** [review]
After some discussion with @open-telemetry/docs-maintainers, we've agreed to accept this "patch" PR. I've added a note to that effect to the `default_lang_commit` front-matter line.

Thanks for the PR @emdneto, and for your patience as we iron out protocol relative to i18n PRs.

**Comment by chalin** [comment]
The i18n check is failing. Let me see if I can fix the checker to support comments on the same line as the hash.

**Pull request merged**
